articles/resources/relnotes.md

```diff
@@ -17,6 +17,16 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version X.XX.XXXXXXXXXX
+
+*Release date: January 26th, 2021*
+
+- Simplified qubit allocation, providing more convenient syntax for allocating qubits, [see details in Q# language repository](https://github.com/microsoft/qsharp-language/blob/main/Approved/1-implicitly-scoped-qubit-allocation.md).
+- Improved the chemistry library including API improvements and better Python support and integration with existing chemistry packages.
```

Comment:
This makes it sound like improvements to the Q# API surface, since we've mainly used the term "chemistry library" in that sense (somewhat inaccurately, given the significant C# API surface to that library in particular, but I digress). The improvements made are (@guenp, please correct me if I'm wrong) provided as a new Python library that _does not depend on Q# at all_ for generating problem descriptions.